docs/management/admin/response-actions-config.asciidoc

```diff
@@ -20,7 +20,7 @@ Check out <<third-party-actions>> to learn which response actions are supported
 --
 * https://www.elastic.co/pricing[Subscription level]: Enterprise 
 
-* {kibana-ref}/kibana-role-management.html#adding_kibana_privileges[{kib} feature privilege]: **Actions and Connectors : All**.
+* {kibana-ref}/kibana-role-management.html#adding_kibana_privileges[{kib} feature privilege]: Under **Actions and Connectors**, turn on **Customize sub-feature privileges** and enable **Endpoint Security**.
```

Comment:
Minor nit in case users are selecting a checkbox to enable Endpoint Security. If they aren't, ignore this!

```suggestion
* {kibana-ref}/kibana-role-management.html#adding_kibana_privileges[{kib} feature privilege]: Under **Actions and Connectors**, turn on **Customize sub-feature privileges** and select **Endpoint Security**.
```